pkgs/desktops/cinnamon/nemo/default.nix

```diff
@@ -0,0 +1,32 @@
+{ fetchFromGitHub, glib, gobject-introspection, meson, ninja, pkgconfig, stdenv, wrapGAppsHook, libxml2, cmake, gtk3, libnotify, cinnamon-desktop, xapps, libexif, exempi, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "nemo";
+  version = "4.4.1";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "0sskq0rssxvna937md446x1489hkhxys1zq03hvl8asjqa259w2q";
+  };
+
+  patches = [ ./0001-patch-datadir.patch ];
+
+  preInstallPhases = "exportDestdir";
+
+  exportDestdir = "export DESTDIR=$out";
```

Comment:
DESTDIR is not a good idea https://github.com/NixOS/nixpkgs/issues/65718

Reply:
It's used in an if statement to determine whether or not update-mimedb should be run by some script. Should I add a comment or add a patch?

Reply:
Comment on anything out of ordinary is a good idea.

Reply:
Looks like it breaks stuff anyways, made a patch to remove script instead